src/wallet/bdk_wallet/wallet.rs

```diff
@@ -25,3 +36,191 @@ pub struct Wallet {
     rpc: Client,
     wallet_file_path: PathBuf,
```

Comment:
its should be called the `data_dir`. Wallet file path is ambiguous. There are two wallet related files, which one is this talking about?